src/App.tsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const getFiltredQuery = () => (query
+    ? moviesFromServer.filter(movie => {
+      const title = movie.title.toLowerCase();
+      const description = movie.description.toLowerCase();
+
+      return title.includes(query.trim().toLocaleLowerCase())
+      || description.includes(query.trim().toLocaleLowerCase());
```

Comment:
create a variable for `query.trim().toLocaleLowerCase())` to not repeat yourself